### PR TITLE
Fix issue with null reason

### PR DESCRIPTION
**Pull request by RishatShamsutdinov**
The fix for https://github.com/firebase/firebase-ios-sdk/issues/8671#event-5344539601

### CODE REVIEW

**Comment by RishatShamsutdinov** [comment]
@googlebot I signed it

Чт, 23 сент. 2021 г. в 12:27, google-cla[bot] ***@***.***>:

**Comment by RishatShamsutdinov** [comment]
I've checked and an exception with empty non-null reason does appear on the dashboard.

File: Crashlytics/Crashlytics/Handlers/FIRCLSException.mm [hunk 1]
```diff
@@ -80,7 +80,7 @@ void FIRCLSExceptionInitialize(FIRCLSExceptionReadOnlyContext *roContext,
 
 void FIRCLSExceptionRecordModel(FIRExceptionModel *exceptionModel) {
   const char *name = [[exceptionModel.name copy] UTF8String];
-  const char *reason = [[exceptionModel.reason copy] UTF8String];
+  const char *reason = [[exceptionModel.reason ?: @"" copy] UTF8String];
```
Comment (morganchen12) [inline Crashlytics/Crashlytics/Handlers/FIRCLSException.mm:83]: nit: Can this be
```objc
const char *reason = [[exceptionModel.reason copy] UTF8String] ?: "";
```
instead?

Comment (RishatShamsutdinov) [reply Crashlytics/Crashlytics/Handlers/FIRCLSException.mm]: Sure, done

**Review by morganchen12 — COMMENTED** [review]
LGTM but deferring to @samedson for approval.

**Review by samedson — APPROVED** [review]

**Pull request merged**